finetuning/generalists/training/light_microscopy/train_lm_generalist.py

```diff
@@ -0,0 +1,97 @@
+import os
```

Comment:
I am confused why these files are part of this PR, they should have been merged via #228 already!

Reply:
Same as above